configs/virtual.js

```diff
@@ -23,7 +23,11 @@ store.put = (k, v) => { s[k] = v; return Promise.resolve(null) }
 store.del = (k) => { s[k] = undefined; return Promise.resolve(null) }
 
 const crypto = require('crypto')
-const token = crypto.randomBytes(32).toString('hex')
+const base64url = require('base64url')
+const token = base64url(JSON.stringify({
+  channel: crypto.randomBytes(32).toString('hex'),
+  host: 'ws://broker.hivemq.com:8000'
+}))
```

Comment:
That part is just to give the plugin a well formed token. The connection is mocked so it won't connect to hivemq. I'll change it to an example site to be clearer.